van-gogh-gallery/josefin/pages/about.css

```diff
@@ -0,0 +1,191 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap');
+
+* {
+    margin: 0px;
+    padding: 0px;
+    font-family: 'Montserrat', sans-serif;
+    
+}
+
+.nav-container{
+    position: absolute;
+   
+    margin-left: 0px;
+    margin-bottom: 10px;
+    cursor: pointer;
+    transition: all 0.2s;
+    font-size: 20px;
+font-weight: bolder;
+justify-content: space-evenly;
+}
+
+li{
+    display: inline-block;
+
+}
+li a{
+  text-decoration: none; 
+  color: black;
+  font-family: 'Montserrat', sans-serif;; 
+  font-weight: lighter ;
+  font-size: 22px;
+  padding: 0 20px;
+  padding-left: 40px;
+  font-weight: 900;
+}
+
+.nav-link::after {
```

Comment:
I think we don't use this after element anywhere 🤔

Reply:
oh i think i tried somethin out and then forgot to delete all of it :o